internal/repository/account_repo_psql.go

```diff
@@ -0,0 +1,95 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/gopay/internal/models"
+)
+
+const (
+	createQ = `
+	INSERT INTO accounts 
+	(name, last_name) 
+	VALUES ($1, $2) 
+	RETURNING account_id
+	`
+
+	findAllQ = `
+	SELECT * 
+	FROM accounts
+	`
+
+	findOneQ = `
+	SELECT *
+	FROM accounts
+	WHERE account_id = $1
+	`
+)
+
+type AccountRepoPsql interface {
+	FindAll(ctx context.Context) ([]models.Account, error)
+	FindOne(ctx context.Context, id string) (models.Account, error)
+	Create(ctx context.Context, name string, lastname string) (string, error)
+}
+
+var _ AccountRepoPsql = (*accountRepoPsqlImpl)(nil)
+
+type accountRepoPsqlImpl struct {
+	psql *sql.DB
+}
+
+func NewAccountRepoPsql(db *sql.DB) *accountRepoPsqlImpl {
+	return &accountRepoPsqlImpl{
+		psql: db,
+	}
+}
+
+func (r *accountRepoPsqlImpl) FindAll(_ context.Context) ([]models.Account, error) {
+	accs := []models.Account{}
+
+	rows, err := r.psql.Query(findAllQ)
+	if err != nil {
+		return accs, err
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		acc := models.Account{}
+		rows.Scan(&acc.AccountId, &acc.Name, &acc.LastName)
+		accs = append(accs, acc)
```

Comment:
For your query, you should specify the order of the fields, so you can capture them in this exact order that you are scanning.

It makes easier match the scanning with the query, and it is safer in case you change the table schema, the query will still preserve the correct order of the attributes that you are scanning